config/manager/manager.yaml

```diff
@@ -22,14 +22,18 @@ spec:
           args:
             - "--leader-elect"
             - "--v=2"
-            - "--metrics-bind-addr=127.0.0.1:8080"
+            - "--diagnostics-address=${CAPO_DIAGNOSTICS_ADDRESS:=:8443}"
+            - "--insecure-diagnostics=${CAPO_INSECURE_DIAGNOSTICS:=false}"
```

Comment:
Until we've sorted out how we want to expose secure metrics, please can we put this back to the old, insecure. Also I'd prefer not to add new cluster api variables for substitution in this change. Lets do that separately and deliberately when we understand this feature.

For now we could either:
* Leave it as it is. `--metrics-bind-addr` is deprecated but still works and has the same behaviour as before.
* Change it to `--diagnostics-address=localhost:8080 --insecure-diagnostics=true`, which keeps the same behaviour but without using the deprecated flag.

My feeling is that the prudent option is to entire revert the change to this file, leaving it exactly as it was before, and open an issue to track adding support for secure metrics.